src/views/List.jsx

```diff
@@ -1,25 +1,29 @@
 import { ListItem } from '../components';
-import { useState } from 'react';
+import React, { useState } from 'react';
 import { AddItems } from '../components/AddItems';
 import TextInputElement from '../components/TextInputElement';
 import { useEnsureListPath } from '../hooks/useEnsureListPath';
+import { useUrgency } from '../utils';
 
-export function List({ data, listPath }) {
+export const List = React.memo(function List({ data, listPath }) {
```

Comment:
Love for you to highlight why we're memorizing the list here!

Reply:
I apologize I actually do not have any description added, but it was re-rendering a bunch of times even before calculating any urgency, and adding this memo helped to reduce the amount of unnecessary re-renders. I will add a short comment on top of the List component